### ISSUE TITLE
Please publish hash/checksum for releases

### ISSUE THREAD

**Issue by rfay** [drush-ops/drush#4744]
**Is your feature request related to a problem? Please describe.**

Currently there's no way to validate a download of drush.phar from the releases page.

**Describe the solution you'd like**

Provide a file hash so the download can be verified.

**Additional context**

Supply chain exploits are the order of the day. We're trying to make sure that ddev downloads trusted things in each of its upstream sources, and drush is one of those. Beyond that though, drush itself could be a key target for a supply-chain exploit, so it's just an obvious thing to do.

Thanks!

**Comment by greg-1-anderson** [comment]
Is this request for Drush 8? Drush 10 no longer distributes a .phar.

Drush 8 releases are infrequent. If we were to provide a file hash, how would you recommend it be published?

**Comment by rfay** [comment]
Yes, drush8. ddev supports D6/D7, so drush8 is installed in /usr/local/bin for those. 

Many releases pages publish a hash for each download; ddev does that, not sure if there are better ways than it's done there, see the .sha256.txt files in https://github.com/drud/ddev/releases/tag/v1.17.1

Open to any way that allows validation of download contents.
